builtin/mainmenu/dlg_contentstore.lua

```diff
@@ -432,20 +432,15 @@ function store.get_formspec()
 		formspec = {
 			"size[12,7;true]",
 			"position[0.5,0.55]",
-			"label[4,3;No packages could be retrieved]",
+			"label[4,3;",
+			fgettext("No results"), "]",
 			"button[-0.1,",
 			num_per_page + 1.5,
 			";3,1;back;",
 			fgettext("Back to Main Menu"), "]",
 		}
 	end
 
-	if #store.packages == 0 then
```

Comment:
Is `store.packages` same as `store.packages_full`?

Reply:
No, `store.packages` is filtered where as `store.packages_full` is complete. 
Ie: `#store.packages <= #store_packages_full`

Reply:
The correct fix would be to move this statement to the bottom of the ` if #store.packages_full > 0 then` block

Reply:
Oh I just realized that this label would be used even if the filtered list returns no results, even if the full list has been populated. On it...

Reply:
Oki, done (I think)